static/app/utils/sessions.tsx

```diff
@@ -72,6 +73,31 @@ export function getCrashFreeSeries(
   );
 }
 
+export function getAdoptionSeries(
+  releaseGroups: SessionApiResponse['groups'] = [],
+  allGroups: SessionApiResponse['groups'] = [],
+  intervals: SessionApiResponse['intervals'] = [],
+  field: SessionField
+): SeriesDataUnit[] {
+  return intervals.map((interval, i) => {
+    const intervalReleaseSessions = releaseGroups.reduce(
+      (acc, group) => acc + group.series[field][i],
+      0
+    );
+    const intervalTotalSessions = allGroups.reduce(
+      (acc, group) => acc + group.series[field][i],
+      0
+    );
+
+    const intervalAdoption = percent(intervalReleaseSessions, intervalTotalSessions);
+
+    return {
+      name: interval,
+      value: round(intervalAdoption),
```

Comment:
Can we add a comment why we're using `lodash.round` instead of `Math.round`?

Reply:
We are not doing any fancy rounding, `Math.round` will work just fine. I'll change it 👍